connector/connector.proto

```diff
@@ -121,7 +121,8 @@ message SocketConfig {
   string id = 1;
   string name = 2;
   string type = 3;
-  google.protobuf.Struct config = 4;
+  bool connector_authentication_enabled = 4;
```

Comment:
Do we need this? I thought we dont support disabling this anymore?

Reply:
This is a breaking change, if we want this we should not change the ordering.
But I think it's better to just make this part of the config struct and not an additional field in the SocketConfig.